### ISSUE TITLE
Set up staging vhost for django app

### ISSUE THREAD

**Issue by zarino** [mysociety/neighbourhood-warmth#12]
- neighbourhoodwarmth.staging.mysociety.org ?
- deploying from the `django` branch for now, until https://github.com/mysociety/neighbourhood-warmth/pull/11 is merged into `main`

**Comment by sagepe** [comment]
See also https://github.com/mysociety/sysadmin/issues/1772

**Comment by sagepe** [comment]
Initial very basic site deployed https://neighbourhood-warmth.staging.mysociety.org/

**Comment by zarino** [comment]
Woop! Thanks @sagepe ! Is that currently deploying off the [`deployment`](https://github.com/mysociety/neighbourhood-warmth/pull/20) branch?

**Comment by sagepe** [comment]
> Is that currently deploying off the [`deployment`](https://github.com/mysociety/neighbourhood-warmth/pull/20) branch?

Yes, can be switched to main once #20 is finished

**Comment by sagepe** [comment]
All done, I think.
